CONTENTS.md

```diff
@@ -0,0 +1,423 @@
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### update_crawler
+
+Create SRs for Leap.
+
+* Sources: [update_crawler.py](update_crawler.py) and [script](script).
+* Documentation: --
+* Package: openSUSE-release-tools-leaper
+* Usage: see [leaper](#leaper)
+
+#### create_staging
+
+Scripts and templates to create staging projects.
+
+* Sources: [staging_templates](staging_templates)
+* Documentation: --
+* Package: --
+* Usage: ?
+
+### Bots
+
+#### check_maintenance_incidents
+
+Handles maintenance incident requests
+
+* Sources: [check_maintenance_incidents.py](check_maintenance_incidents.py)
+* Documentation: [docs/maintbot.asciidoc](docs/maintbot.asciidoc)
+* Package: openSUSE-release-tools-maintenance
+* Usage: gocd
+
+#### leaper
+
+Implements Leap-style services for non-Factory projects (whatever that means).
+
+* Sources: [leaper.py](leaper.py)
+* Documentation: --
+* Package: openSUSE-release-tools-leaper
+* Usage: gocd
+
+#### origin-manager
+
+Keeps track of from what project a package originates, submit updates, review requests to detect origin changes, and enforce origin specific policies like adding appropriate reviews
+
+* Sources: [origin-manager.py](origin-manager.py) and [web](web)
+* Documentation: [docs/origin-manager.md](docs/origin-manager.md)
+* Package: openSUSE-release-tools-origin-manager
+* Usage: gocd
+
+#### staging-bot
+
+Assists in management of staging projects. It is composed of Python scripts and a few systemd
+units.
+
+* Sources: [devel-project.py][devel-project], [staging-report.py](staging-report.py), [suppkg_rebuild.py](suppkg_rebuild.py) and `osrt-staging-bot-*` [systemd units](systemd)
+* Documentation: --
+* Package: openSUSE-release-tools-staging-bot
+* Usage: gocd
+
+#### legal-auto
+
+Makes automatic legal reviews based on the legaldb API
+
+* Sources: [legal-auto.py](legal-auto.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### check_tags_in_requests
+
+Checks that a submit request has corrrect tags specified.
+
+* Sources: [check_tags_in_requests.py](check_tags_in_requests.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### abichecker
+
+Checks ABI compatibility in OBS requests.
+
+* Sources: [abichecker](abichecker)
+* Documentation: --
+* Package: openSUSE-release-tools-abichecker
+* Usage: gocd
+
+#### check_source_in_factory
+
+Checks if the sources of a submission are either in Factory or a request for Factory with the same
+sources exist.
+
+* Sources: [check_source_in_factory.py](check_source_in_factory.py)
+* Documentation: [docs/factory-source.asciidoc](docs/factory-source.asciidoc)
+* Package: openSUSE-release-tools
+* Usage: gocd?
+
+#### openqa-maintenance
+
+OpenQA stuff, not sure about the details.
+
+* Sources: [openqa-maintenance.py](openqa-maintenance.py) and [oqamaint](uqamaint)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd?
+
+#### check_tags_in_requests.py
```

Comment:
It is here twice. See https://github.com/openSUSE/openSUSE-release-tools/pull/2604/files#diff-f8803a0cca97397c1fb1e42e1f462e81dc36b8a87efdb79cf1c116f5bc809413R244